src/sign-transaction/SignTransaction.js

```diff
@@ -0,0 +1,33 @@
+class Api {
```

Comment:
Maybe rename to `SignTransaction`, as that is also the file name and thus the _module_ name?